tests/atoms/gaia_data_layer.js

```diff
@@ -45,16 +45,17 @@ var GaiaDataLayer = {
 
   disableBluetooth: function() {
     var bluetooth = window.navigator.mozBluetooth;
-    if (bluetooth.enabled) {
+    if (bluetooth.defaultAdapter.state == 'enabled') {
```

Comment:
I think we need to perform disable() when ['enabling'](https://wiki.mozilla.org/B2G/Bluetooth/WebBluetooth-v2/BluetoothAdapter#BluetoothAdapterState) is happening.